Adyen/UI/Form/Items/Value Pickers/FormRegionPickerItem.swift

```diff
@@ -37,7 +37,7 @@ internal final class FormRegionPickerItemView: BaseFormPickerItemView<Region> {
     override internal func initialize() {
         let stackView = UIStackView(arrangedSubviews: [titleLabel, inputControl])
         stackView.axis = .vertical
-        stackView.spacing = 8
+        stackView.spacing = 5.5
```

Comment:
this is peculiar number 🤔
Is it empirically calculated?

Reply:
Oh yeah. This is funny cause there's a misalignment between the state picker and the zip code field. The easiest solution is to decrease the spacing to 5.5 so both separator lines have the same Y position.

There's no auto layout involved in positioning both separator lines, so we cannot use that to align them.

I know from a design perspective we work on 8 multiples, but for this case it makes sense to break the rule. Open to suggestions tho

Reply:
If it look good - it is on :)